certmanager/generate_cert_test.go

```diff
@@ -197,7 +197,7 @@ func TestGenCert(t *testing.T) {
 		{
 			// a cert that can only be used as client-side cert
 			certOptions: CertOptions{
-				Host:         "istio:bar.serviceaccount.com",
+				Host:         "spiffe://domain/ns/bar/sa/foo",
```

Comment:
This is only for testing. cert generator actually doesn't care the id format. It's the CA who specifies the IDs